make/cpp2asm_support.cmake

```diff
@@ -132,7 +132,14 @@ endif ()
 ##################################################
 # Preprocessor location and flags
 
-if (UNIX OR (WIN32 AND "${CMAKE_GENERATOR}" MATCHES "MSYS Makefiles"))
+if ("${CMAKE_GENERATOR}" MATCHES "Visual Studio")
```

Comment:
Like I keep saying, these are *compiler* flags.  The *generator* is not what should be checked here.  If I use Ninja or NMake Makefiles on Windows I'll still be using the Visual Studio compiler, and your change here will break my build.

Reply:
@derekbruening See my last commit, I already fixed it.